lms/djangoapps/grades/api/views.py

```diff
@@ -153,8 +155,20 @@ def get(self, request, course_id):
         if isinstance(course, Response):
             return course
 
+        if request.user.username == username:
```

Comment:
If it is better to use `not`?

```suggestion
        if request.user.username != username:
            user = User.objects.filter(username=username).first()
        if not user:
            return self.make_error_response(
                    status_code=status.HTTP_404_NOT_FOUND,
                    developer_message='The requested user does not exist.',
                    error_code='no_user'
                )
```

Reply:
@cmltaWt0 Yes this looks cleaner!